landing_page/app/views/microposts/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+    <h2><%= @micropost.title %></h2>
+    <br></br>
+</p>
+
+<p>
+    <%= @micropost.content %>
+</p>
+
+<p>
+    <strong>Created at:</strong>
+    <%= @micropost.created_at %>
+</p>
+
+<p>
+    <strong>By:</strong>
+    <%= User.find_by_id(@micropost.user_id).email%>
+</p>
+
+<p>
+    <strong>Updated at:</strong>
+    <%= @micropost.updated_at %>
+</p>
+
+<br></br>
+<br></br>
+<br></br>
+
+<h6><%= link_to "Back to List of Blog Posts", microposts_path %></h6>
```

Comment:
Here you used `h6` but you don't have any other headings in the page. Headers are used not only for visuals, as in bigger or smaller texts, but also to categorize the contents of the page. Ideally you should always start with a `h1` and continue on organizing the content. Have a look at this article: https://www.w3schools.com/html/html_headings.asp